src/main.rs

```diff
@@ -1169,13 +1169,14 @@ impl IRCBotClient {
             }
             "feature:aaleaderboard" => {
                 self.aa_leaderboard.fetch_if_required().await;
-                let trimmed_args = args.trim(); // get random spaces at end of messages sometimes
+                let trimmed_args = args.trim_matches(|c: char| !c.is_alphanumeric()); // get random characters at end of messages sometimes
```

Comment:
I think this would break for valid MC usernames, like `totorewaxx__`, right?

Reply:
Yeah you're right. There's a weird character that sometimes appears at the end of people's messages. I'm not sure what the character is. Maybe it's a thing 7TV is doing? I'll revert it to trim though. If the user happens to have this character at the end of their message the lookup will fail and they'll just have to type the command again.

Reply:
Reverted

Reply:
oh that is a 7tv thing, I think you can actually check for it, it's like nbsp I think?
they insert a random number of them at the end to get past 'same message spam' warnings

Reply:
Ohhh OK that makes sense. I'll look in to that soon.

Reply:
Now trims non-ascii characters and whitespace